fastly/block_fastly_service_v1_package.go

```diff
@@ -0,0 +1,108 @@
+package fastly
+
+import (
+	"fmt"
+	"log"
+
+	gofastly "github.com/fastly/go-fastly/fastly"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+)
+
+type PackageServiceAttributeHandler struct {
+	*DefaultServiceAttributeHandler
+}
+
+func NewServicePackage() ServiceAttributeDefinition {
+	return &PackageServiceAttributeHandler{
+		&DefaultServiceAttributeHandler{
+			key: "package",
+		},
+	}
+}
+
+func (h *PackageServiceAttributeHandler) Register(s *schema.Resource) error {
+	s.Schema[h.GetKey()] = &schema.Schema{
+		Type:     schema.TypeList,
+		Required: true,
```

Comment:
I don't think this needs to be required. This would prevent me from provisioning an Wasm service and populating the package externally.

Reply:
AFAIK from my experiments, the API bounces any requests to create a Wasm service without a package

Reply:
We discussed this offline. While you can create Wasm services without a package, you need a package in order to validate and activate a service version (which the TF service logic will implicitly try). Therefore we will leave as `required`.